lizmap/modules/view/forms/edition.form.xml

```diff
@@ -9,7 +9,7 @@
 <hidden ref="liz_proj4"/>
 <hidden ref="liz_geometryColumn"/>
 <hidden ref="liz_status"/>
-<hidden ref="_submit"/>
+<hidden ref="_submit"/> <!-- why ? duplicated field -->
```

Comment:
To store the submit value in the form: https://github.com/3liz/lizmap-web-client/commit/4ca94407a71328cf0780284c601fee7e4e339aed#diff-4c07e19f3b7ab2f105b286b0564be51333f7d8ae4f98602921f8f1d49aa3a497R830